lib/tests/fixtures/facts/external/windows/powershell/yaml.ps1

```diff
@@ -0,0 +1,19 @@
+$yaml_content = @'
+---
+PS1_YAML_FACT1: foo
+PS1_yaml_fact2: 2
+ps1_yaml_fact3: |
+  one value
+  but
+  many lines
+ps1_yaml_fact4:
+  - one
+  - two
+ps1_yaml_fact5: [first, second, third]
+ps1_yaml_fact6:
+  red: green
+  orange: blue
+  yellow: purple
+ps1_yaml_fact7: { key: value }
+'@
+Write-Output($yaml_content)
```

Comment:
I think you can do the same here... remove the variable assignment and `Write-Output`, and I believe it should work automatically.